### PR TITLE
Add CI for GitHub Actions | Remove CircleCI in gvm-libs-20.08

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] [CHANGELOG](https://github.com/greenbone/gvm-libs/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

**Comment by y0urself** [comment]
Need to resolve the potential memory leak ...

**Comment by y0urself** [comment]
> Need to resolve the potential memory leak ...

I wonder why this leak appeared in 20.08, as it is "the same" in 21.04 and master ...

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "gvm-libs"
+version = "21.10.0.dev1"
```
Comment (y0urself) [inline pyproject.toml:3]:
```suggestion
version = "20.08.1"
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "gvm-libs"
+version = "20.08.1"
```
Comment (y0urself) [inline pyproject.toml:3]:
```suggestion
version = "20.8.1"
```

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**